servers/bid/server/routes.go

```diff
@@ -21,6 +21,11 @@ func NewRouter(DB *gorm.DB, redisClient *redis.Client) *gin.Engine {
 	{
 		bidGroup.GET("/health", sharedController.HealthCheckHandler())
 		bidGroup.POST("/bid", controller.AddBidHandler(bidUtils))
+		bidGroup.GET("/getbidbyp/:postID", controller.GetBidsForPostHandler(bidUtils))
```

Comment:
rebase to the latest development branch. uncomment the default group. 

for Get operations put in default group

for data modifications PUT POST DELETE put in sensitive group.